### PR TITLE
[DICOM Archive] Fully react-rendered + written in ES6

### PR DESCRIPTION
**Pull request by alisterdev**
🎉 **New cool features below** 🎉 

- [x] Updated `dicom_archive` as an example module written in ES6 
- [x] Simplified code for the Filter, removed unnecessary state
- [x] Completely moved page rendering to Javascript

>**Note 1**
In order for changes above to work, had to update QueryString API, and include it globally. 
>**Note 2**
Sorry for large amount of changed files @jacobpenny 
>**Note 3**
This update relies on https://github.com/aces/Loris/pull/2433, so it shouldn't be merged before 2433

### CODE REVIEW

File: jsx/LorisPage.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
+/* global LorisPage */
+/* global QueryString  */
+
+const propTypes = {};
+const defaultProps = {};
```
Comment (jacobpenny) [inline jsx/LorisPage.js]: Might as well define these now

Comment (alisterdev) [inline jsx/LorisPage.js]: @jacobpenny The default properties are page specific

Comment (jacobpenny) [inline jsx/LorisPage.js]: Ah gotcha, makes sense.

**Comment by jacobpenny** [comment]
I think it is worth reconsidering the use of inheritance here because it isn't idiomatic in the React world. Facebook has a pretty strong opinion on this: https://facebook.github.io/react/docs/composition-vs-inheritance.html#so-what-about-inheritance

Also summed up nicely by one of the core React contributors here:
https://github.com/facebook/react/issues/613#issuecomment-29280175

**Comment by alisterdev** [comment]
@jacobpenny Scratched the whole inheritance idea as per your suggestion and simplified filter interface instead.

**Pull request merged**